perf_test.go

```diff
@@ -89,6 +181,26 @@ func TestPerfReaderLostSample(t *testing.T) {
 	}
 }
 
+// TODO - xdpcap.Hook.Patch() uses the same thing, should we add a CollectionSpec.UseMap?
+// Or maybe just ProgramSpec.RewriteMap(), and have NewCollection() not rewrite already rewritten maps.
```

Comment:
We've talked about this before, but I can't remember the conclusion.

Reply:
Yeah, I think it makes sense for RewriteMap to return a new error if insn.Constant != 0, and then catch that in NewCollection. See my comment above though, maybe we don't need this?

Reply:
Yup, turns out we don't need this at all. Sounds sane enough :)